Automattic/wp-calypso
# Help Center: link to support docs if disconnected from JPOP

Fixes #91558 

## Proposed Changes

Add check to Help Center in ETK to determine if user is connected to Jetpack. Then load simplified version if not.

## Why are these changes being made?

Help Center relies on a WordPress.com user to do most of its functions. For Atomic and Jetpack only users this means they need to be connected to WordPress.com in order for it to work. To prevent loading a useless Help Center and save on loading times we have decided it would be best to make Help Center a button that opens the support docs if the user is not connected.

## Testing Instructions

1. Download the ETK plugin and install it on an atomic site.
2. Go to `/wp-admin` and the post editor to make sure the help center is functioning as normal.
3. Disconnect from Jetpack. I did this by using the `/_cli` and running `jetpack disconnect user ##USER_ID## --force`. Be aware that this will remove your SSH key connection to the site if you have it setup.
4. Go to `/wp-admin` and the post editor to see the Help Center open a new tab to the support docs.
5. Reconnect Jetpack
